### PR TITLE
[ADD] project_task_markdown

### PR DESCRIPTION
**Pull request by KKamaa**
Add markdown to description field in project task as an option from default odoo version 13.0 base on [web_widget_text_markdown](https://github.com/OCA/web/pull/1965)

### CODE REVIEW

**Comment by KKamaa** [comment]
@thomaspaulb This has been migrated, added the branch repo to oca-requirements.txt as stated in[ doc](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#185oca_dependenciestxt). Travis finds an error in another module not related to this ADD, not sure why.

File: project_task_markdown/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    description = fields.Char(string="Description")
```
Comment (thomaspaulb) [inline project_task_markdown/models/project_task.py]: `fields.Text` seems more logical?

File: project_task_markdown/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* Tom Blauwendraat <tblauwendraat@therp.nl>
```
Comment (thomaspaulb) [inline project_task_markdown/readme/CONTRIBUTORS.rst]: tom@sunflowerweb.nl

**Review by thomaspaulb — CHANGES_REQUESTED** [review]
Hehe... seems you have just found a timezone bug in `project_timesheet_time_control` :-) Since you like to work around midnight, that's when bugs happen if date functions do not correctly take into account the time zone.

**Comment by thomaspaulb** [comment]
Anyway that test is not ours to fix.. and if you push a fix around daytime I think Travis will be green.

File: project_task_markdown/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (KKamaa) [reply project_task_markdown/readme/CONTRIBUTORS.rst]: fixed

File: project_task_markdown/models/project_task.py (hunk 1 above)
Comment (KKamaa) [reply project_task_markdown/models/project_task.py]: This works too, pushed the fix

**Review by thomaspaulb — APPROVED** [review]

**Comment by gjotten** [comment]
First look on a running instance is promising but I'd like to do some more functional tests before final approval :-)

**Review by ajaniszewska-dev — APPROVED** [review]

**Pull request merged**